.gitignore

```diff
@@ -1,3 +1,4 @@
 /target
 /vendor/boost
 /touchHLE_sandbox
+.idea
```

Comment:
Was this added automatically by your IDE? I think there's some way to tell git to ignore files locally, without having to modify the repo.

Reply:
It was added by me, because CLion puts some stuff in `.idea` (and if somebody will try to open this in CLion they may accidentally commit some garbage). I can remove it and put it to my local .gitignore if you want though

Reply:
I'd prefer that since this might end up being a laundry list of IDEs otherwise, which is a distraction from the main purpose IMO.

Reply:
>  laundry list of IDEs

It often is, just look at https://github.com/dotnet/runtime/blob/main/.gitignore

Reply:
I guess it's a good idea if you have a large project and a lot of novice contributors, but I feel like it's a workaround for people not being familiar with better tools, to be honest. I'd rather keep the `.gitignore` to stuff specific to this project.